applications/picocdn-node-vm/templates/cdn-service.yaml

```diff
@@ -0,0 +1,30 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2021 NetPico LAbs Pvt. Ltd.(picoNETS)
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "picocdn-node-vm.fullname" . }}-service
+spec:
+  type: {{ default "NodePort" .Values.cdnService.type }}
+  selector:
+    kubevirt.io/domain: {{ template "picocdn-node-vm.fullname" . }}
+  externalIPs:
+    192.168.1.100
```

Comment:
'-' sign needs to be added before IP. 
for eg: https://kubernetes.io/docs/concepts/services-networking/service/#external-ips

Reply:
Fixed this yaml formatting error, also added the comment identifying that this is an example IP that needs to be updated prior to helm chart deployment. We already had this item in our Installation section of the Readme.md, so this provides the glue instructions in the file.